[2022-10-04 13:28:27,248] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ingest_star_wars.get_raw_star_wars_films_data manual__2022-10-04T13:28:23.237734+00:00 [queued]>
[2022-10-04 13:28:27,263] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ingest_star_wars.get_raw_star_wars_films_data manual__2022-10-04T13:28:23.237734+00:00 [queued]>
[2022-10-04 13:28:27,264] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-10-04 13:28:27,265] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-10-04 13:28:27,266] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-10-04 13:28:27,285] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): get_raw_star_wars_films_data> on 2022-10-04 13:28:23.237734+00:00
[2022-10-04 13:28:27,291] {standard_task_runner.py:52} INFO - Started process 3804 to run task
[2022-10-04 13:28:27,297] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'ingest_star_wars', 'get_raw_star_wars_films_data', 'manual__2022-10-04T13:28:23.237734+00:00', '--job-id', '34', '--raw', '--subdir', 'DAGS_FOLDER/ingest_starwars_data.py', '--cfg-path', '/tmp/tmpwjpdwwvb', '--error-file', '/tmp/tmpm1ld_0ih']
[2022-10-04 13:28:27,299] {standard_task_runner.py:77} INFO - Job 34: Subtask get_raw_star_wars_films_data
[2022-10-04 13:28:27,375] {logging_mixin.py:109} INFO - Running <TaskInstance: ingest_star_wars.get_raw_star_wars_films_data manual__2022-10-04T13:28:23.237734+00:00 [running]> on host be1a5b89e146
[2022-10-04 13:28:27,440] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=xxxx@xxxxx.com
AIRFLOW_CTX_DAG_OWNER=Ram
AIRFLOW_CTX_DAG_ID=ingest_star_wars
AIRFLOW_CTX_TASK_ID=get_raw_star_wars_films_data
AIRFLOW_CTX_EXECUTION_DATE=2022-10-04T13:28:23.237734+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-10-04T13:28:23.237734+00:00
[2022-10-04 13:28:30,761] {python.py:175} INFO - Done. Returned value was: None
[2022-10-04 13:28:30,778] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=ingest_star_wars, task_id=get_raw_star_wars_films_data, execution_date=20221004T132823, start_date=20221004T132827, end_date=20221004T132830
[2022-10-04 13:28:30,848] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-10-04 13:28:30,924] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
